content/22.vaccines.md

```diff
@@ -448,6 +448,28 @@ Novavax-CoV2373 was able to induce a multifunctional CD4/CD8 T-cell responses an
 These findings were subsequently evaluated in a baboon primate model, in which Novavax-CoV2373 also elicited high antibody titers against the SARS-CoV-2 spike protein, as well as an antigen specific T-cell response.
 Based on this data Novavax initiated a Phase 1/2 clinical trial to evaluate the safety and immunogenicity of Novavax-CoV2373 with Matrix-M (NCT04368988)[@doi:10.1056/NEJMoa2026920].
 
+The phase 1/2 trial was a randomized, placebo-controlled study with 131 healthy adult participants in 5 treatment arms.
+Participants that received the recombinant SARS-CoV-2 vaccine with or without Matrix-M adjuvant got two injections, 21 days apart.
+Primary outcomes that were assessed include reactogenicity, lab-values (serum chemistry and hematology) as well as anti-spike IgG levels.
+Secondary outcomes measured included virus neutralization, T-cell responses, and unsolicited adverse events.
+The authors reported that no treatment-related adverse serious events occurred in any of the treatment arms.
+Reactogenicity was mostly absent and of short duration.
+The two-dose vaccine regimen induced anti-spike IgG levels and neutralizing antibody-titers exceeding those in convalescent plasma of symptomatic patients.
+In line with the preclinical studies, the use of Matrix-M adjuvant further increased anti-spike immunoglobulin levels and induced a Th1 response.
+The outcomes of this trial suggest that Novavax-CoV2373 has an acceptable safety profile and is able to induce a strong immune response with high neutralizing antibody titers.
+The phase 2 component of this phase 1/2 trial was recently uploaded to an open-access repository [@doi:10.1101/2021.02.26.21252482].
+This part of the trial was designed to identify which dosing regimen should move forward into late phase clinical trials.
+Both younger (18-59 years) and older patients (60-84 years) were randomly assigned to receive either 5ug or 25ug Novavax-CoV2373 or placebo in two doses, 21 days apart.
+In line with the phase 1 data, reactogenicity remained mild to moderate and of short duration. 
+Both doses were able to induce high anti-spike IgG titers as well as neutralizing antibody responses after the second dose.
+Based on both safety and efficacy data, the 5 ug dosing regimen was selected as the optimal dose regiment for the ongoing phase 3 trial.
+Although the phase 3 trial data has not been published yet, Novavax announced an efficacy of 89.3% based on their phase 3 trial in the UK and South Africa.
```

Comment:
Is there a press release or website we can cite for this?

Reply:
Same for the phase 1 data :)

Reply:
Yes: https://ir.novavax.com/news-releases/news-release-details/novavax-covid-19-vaccine-demonstrates-893-efficacy-uk-phase-3